src/lib/pixel-select-drop-down/pixel-select-drop-down.tsx

```diff
@@ -211,6 +218,47 @@ export const PixelDropDown = React.forwardRef<HTMLDivElement, DropDownProps>(
     const truncate = (label, max) => {
       return label?.length > max ? label.substr(0, max - 1) + '...' : label
     }
+
+    const [selectedOption, setSelectedOption] = React.useState([])
+
+    const handleRadioClick = (option) => {
+      rest.onChange &&
+        rest.onChange({
+          target: {
+            value: option.value,
+            options: [{ text: option.label }],
+            selectedIndex: 0
+          }
+        })
+      setIsOptionsOpen(isShowRadio ? true : false)
```

Comment:
no need of adding `? true : false` this condition 
if `isShowRadio` is true it is set true automatically in `setIsOptionsOpen`